### PR TITLE
package-lock.json: run npm dedupe

### PR DESCRIPTION
**Pull request by kevinburke**
I want to do this independently of modifying the package.json file so
that diffs can be evaluated more easily. This was generated by running
"npm dedupe" only with npm 6.4.1.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1811,6 +1675,15 @@
       "integrity": "sha1-FopHAXVran9RoSzgyXv6KMCE7WM=",
       "dev": true
     },
+    "combined-stream": {
```
Comment (kevinburke) [inline package-lock.json]: Please double check this is the correct/a safe version of this dependency!

**Comment by evilebottnawi** [comment]
@kevinburke why need dedupe? Other people can't have this?

**Comment by kevinburke** [comment]
We don't need it, but it seems like a good idea to not install more dependency versions/take up more space on disk than is necessary, and a smaller package-lock.json file will be easier to evaluate than a larger one.

**Comment by evilebottnawi** [comment]
Not required

**Pull request closed without merging**